src/test/java/marquez/client/MarquezClientTest.java

```diff
@@ -240,10 +245,12 @@ public void testCreateDataset() throws Exception {
     final URL url = buildUrlFor(path);
     when(http.url(pathTemplate, NAMESPACE_NAME)).thenReturn(url);
 
-    final DatasetMeta meta =
-        DatasetMeta.builder()
+    final DbTableMeta meta =
+        DbTableMeta.builder()
+            .type(DatasetType.DB_TABLE)
             .name(DATASET_NAME)
-            .datasourceUrn(DATASOURCE_URN)
+            .physicalName(DATASET_PHYSICAL_NAME)
+            .datasourceName(DATASOURCE_NAME)
```

Comment:
_Minor:_ The `@SuperBuilder` is super handy, but in the future we'll (eventually) want to write our own to have a more concise builder implementation. That is, as a user, if I instantiate the class `DbTableMeta` using the builder, I shouldn't also have to provide it's `type` (i.e. `DB_TABLE`). It should be set on the users behalf.

For example:

```java
    final DbTableMeta meta =
        DbTableMeta.builder()
            .name(DATASET_NAME)
            .physicalName(DATASET_PHYSICAL_NAME)
            .datasourceName(DATASOURCE_NAME)
            .description(DESCRIPTION)
            .build();
```

Reply:
That's a really good point. I'll take another look.

Reply:
I've removed SuperBuilder annotation and am just using Builder on private constructors for StreamMeta and DbTableMeta. 
I think this is inline with what you envisioned.

Reply:
Sweet! Yeah, the builders are a bit more intuitive now 👍